looprpc/client.proto

```diff
@@ -1440,3 +1478,220 @@ message InstantOut {
     */
     string sweep_tx_id = 5;
 }
+
+message NewStaticAddressRequest {
+    /*
+    The client's public key for the 2-of-2 MuSig2 taproot static address.
+    */
+    bytes client_key = 1;
+}
+
+message NewStaticAddressResponse {
+    /*
+    The taproot static address.
+    */
+    string address = 1;
+
+    /*
+    The CSV expiry of the static address.
+    */
+    uint32 expiry = 2;
+}
+
+message ListUnspentDepositsRequest {
+    /*
+    The number of minimum confirmations a utxo must have to be listed.
+     */
+    int32 min_confs = 1;
+
+    /*
+    The number of maximum confirmations a utxo may have to be listed. A zero
+    value indicates that there is no maximum.
+    */
+    int32 max_confs = 2;
```

Comment:
Why is maximum number of confirmation needed? Is it related to CSV? (Exclude expired or about to expire UTXOs.)